index.js

```diff
@@ -0,0 +1,9 @@
+// Toogle Mobile Menu
+const menu = document.querySelector(".mobileMenu");
+const mobileMenu = document.querySelector(".menu-mobile");
+const toggleMenu = () => {
+  mobileMenu.classList.toggle("showMenu");
+};
+
+menu.addEventListener("click", () => toggleMenu());
+mobileMenu.addEventListener("click", () => toggleMenu());
```

Comment:
- [OPTIONAL] To keep your code dry, you could use a forEach loop to loop through n array of menu, mobileMenu and assign them the same event handler. What do you think 🤔 ?

Reply:
hi @wayungi,

Thank you for the feedback, that's a good idea, I will give it a shot.

Thank you.